src/UI/Implementation/Component/Input/Field/TextArea.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/* Copyright (c) 2017 Jesús lópez <lopez@leifos.com> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Input\Field;
+
+use ILIAS\UI\Component as C;
+
+/**
+ * This implements the textarea input.
+ */
+class TextArea extends Input implements C\Input\Field\TextArea {
+
+	/**
+	 * @inheritdoc
+	 */
+	protected function isClientSideValueOk($value) {
+		return is_string($value);
+	}
+
+
+	/**
+	 * @inheritdoc
+	 */
+	protected function getConstraintForRequirement() {
+		return $this->validation_factory->hasMinLength(1);
```

Comment:
Is a length of 1 always enough to fulfill a required textarea? Maybe the component needs a parameter for that purpose, but I'm not really sure.

Reply:
Yes, I think you are right. Textareas can be limited also with a maximum number of characters. I will create the constraint "hasMaxLenght" and deal with the proper parameters to set the values.

Reply:
nice.